LondonBritishLibrary/orient/BLorient768.xml

```diff
@@ -31,21 +31,391 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.6">
+<locus from="25r"/>
+<title xml:lang="gez" type="complete">ሳድስ፡ ዜና፡</title>
+<incipit xml:lang="gez">ኮነ፡ ኦአኃውየ፡ ምእመናኒሁ፡ ለክርስቶስ፡ ያስተርክብክሙ፡ እግዚአብሔር፡ ለፈቃዱ፡ ወይምራኅክሙ፡ ኀበ፡ ሥምረቱ፡ ውስተ፡ ምኔት፡
+ዘይትአመር፡ ምኔተ፡ አባ፡ ጳኵሚስ፡ ዘውስተ፡ አድያመ፡ ፅዒድ፡ መነኮስ፡ ቅዱስ።</incipit>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.7">
+<locus from="29v"/>
+<title xml:lang="gez" type="complete">ሳብዕ፡ ዜና፡ ዘ፩እምቅዱሳን።</title>
+<incipit xml:lang="gez">ኮነ፡ ውስተ፡ ምኔት፡ እምኔታተ፡ መነኮሳት፡ እምአድያመ፡ ሀገር፡ ዘትሰመይ፡ ሳን፡ ቅዱስ፡ ዘይሰመይ፡ ስሙ፡ ዩሴፍ።</incipit>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.8">
+<locus from="34r"/>
+<title xml:lang="gez" type="complete">ሳምን፡ ዜና፡</title>
+<incipit xml:lang="gez">ኮነ፡ ኦአኃውየ፡ ውስተ፡ ደብር፡ ሠርቃዊ፡ ዘእምድረ፡ ግብጽ፡ ዘቅሩብ፡ እምባሕረ፡ ኤርትራ፡ ጽሙድ፡ ዘይነብር፡ ውስቴቱ፡</incipit>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.9">
+<locus from="38r"/>
+<title xml:lang="en" type="complete">Anecdote narrated by the patriarch ʾabbā Benjamin</title>
+<incipit xml:lang="gez">ይቤ፡ አቡነ፡ አብ፡ ሊቀ፡ ጳጳሳት፡ አባ፡ ብንያሚ፡</incipit>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.10">
+<locus from="42r"/>
+<title xml:lang="en" type="complete">Anecdote narrated by ʾabbā Victor, abbat of the convent of Māhw or
+ʾel-Ḥabṭu</title>
+<incipit xml:lang="gez">ዘ፩እምቅዱሳን፡ እምዘ፡ ጸሐፎ፡ አብ፡ ቅዱስ፡ አባ፡ ፊቅጦር፡ ሊቀ፡ ምኔት፡ ዘእልሐብጡ፡ ወይትዓወቅ፡ ምኔተ፡ ማኅው፡ ዘውስተ፡
+ምድረ፡ ሀገረ፡ እለእስክንድርያ።</incipit>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.11">
+<locus from="45r"/>
+<title xml:lang="en" type="complete">Another anecdote narrated by ʾabbā Victor</title>
+<incipit xml:lang="gez">ይቤ፡ ፊቅጦር፡ ጸሓፊ፡ እንዘ፡ አነ፡ ፩ዕለተ፡ ውስተ፡ ማኅበረ፡ አኃው፡ ውስተ፡ ዝንቱ፡ ምኔት።</incipit>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.12">
+<locus from="48r"/>
+<title xml:lang="en" type="complete">Another</title>
+<incipit xml:lang="gez">ዘዘከሮ፡ ፊቅጦር፡ ካዕበ፡ ዘዘንቱ፡ ምኔት።</incipit>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.13">
+<locus from="49r"/>
+<title xml:lang="en" type="complete">Another</title>
+<incipit xml:lang="gez">እምነ፡ ዘዘከሮ፡ ፊቅጦር፡ ካዕበ፡</incipit>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.14">
+<locus from="52r"/>
+<title xml:lang="en" type="complete">Another</title>
+<textLang mainLang="gez">እምዘ፡ ዘከሮ፡ ፊቅጦር፡ ዓዲ።</textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.15">
+<locus from="54r"/>
+<title xml:lang="gez" type="complete">ዘ፩እመነኮሳት፡ ኮነ፡ መነኮስ፡ ይነብር፡ ውስተ፡ ምኔተ፡ ቅዱስ፡ እንጦንስ፡ ዘውስተ፡ ደብር፡ ሠረቃዊ።</title>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.16">
+<locus from="54v"/>
+<title xml:lang="gez" type="complete">ዘ፩እመነኮሳት፡ ቅዱሳን፡</title>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.17">
+<locus from="58r"/>
+<title xml:lang="en" type="complete">Anecdote narrated by Macarius</title>
+<incipit xml:lang="gez">ዘመቃራ፡<sic/> ጸሓፊ። ይቤ፡ መቃሪ፡ ጸሓፊ፡</incipit>
```

Comment:
please, use the opening and closing tags for sic

Reply:
Done